inc/saiswitch.h

```diff
@@ -2265,6 +2265,18 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_TUNNEL_LOOPBACK_PACKET_ACTION,
 
+    /**
+     * @brief Start Firmware Broadcast
+     *
+     * Configure PHY to start Firmware Broadcast.
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @validonly SAI_SWITCH_ATTR_FIRMWARE_DOWNLOAD_BROADCAST == true
+     */
+    SAI_SWITCH_ATTR_FIRMWARE_BROADCAST_START,
```

Comment:
_START and _END ending names for attributes is reserved for ranges, current logic is expecting same attribute with _END naming, maybe change attr name ?